tests/conftest.py

```diff
@@ -95,6 +101,50 @@ def test_something(elasticsearch):
     build_index.delete(ignore=404)
 
 
+@runif_bigquery_testing_enabled
```

Comment:
Does this need to be part of the fixture, when it's already applied to the consumers of the fixture?

Reply:
I checked, and it doesn't need to be.